cmd/topicmappr/commands/rebuild_steps.go

```diff
@@ -117,8 +117,8 @@ func printChangesActions(cmd *cobra.Command, bs *kafkazk.BrokerStatus) {
 	ol, _ := cmd.Flags().GetBool("optimize-leadership")
 
 	// Print broker change summary.
-	fmt.Printf("%sReplacing %d, added %d, missing %d, total count changed by %d\n",
-		indent, bs.Replace, bs.New, bs.Missing+bs.OldMissing, change)
+	fmt.Printf("%sReplacing %d, added %d, missing %d, rack missing %d total count changed by %d\n",
+		indent, bs.Replace, bs.New, bs.Missing+bs.OldMissing, bs.RackMissing, change)
```

Comment:
Debatable whether to include this in the broker change summery since a missing rack is a broker status rather than a change in the brokers we have vs the brokers we want

Reply:
yes, makes sense. I will just remove it as we already print the broker with the missing rack.id plus WARN it.